docs/source/operation/backup.md

```diff
@@ -250,189 +233,110 @@ There are multiple methods for backing up the Gluu Server. A few recommended str
-                kubectl get cm gluu-config-cm -n gluu -o yaml > gluu_cms.yaml && \
-                echo "---" >> gluu_cms.yaml && \
-                kubectl get cm gluu-config-gen-json-file -n gluu -o yaml >> gluu_cms.yaml && \
-                echo "---" >> gluu_cms.yaml && \
-                kubectl get cm gluu-config-tls-script -n gluu -o yaml >> gluu_cms.yaml && \
-                echo "---" >> gluu_cms.yaml && \
-                kubectl get cm gluu-updatelbip -n gluu -o yaml >> gluu_cms.yaml
-                ```
-            
-            1. Save important gluu `Secret`s:
-            
-                ```bash
-                kubectl get secret gluu -n gluu -o yaml > gluu_main_secret.yaml && \
-                kubectl get secret tls-certificate -n gluu -o yaml > gluu_secrets.yaml && \
-                echo "---" >> gluu_secrets.yaml && \
-                kubectl get secret gluu-jackrabbit-admin-pass -n gluu -o yaml >> gluu_secrets.yaml && \
-                echo "---" >> gluu_secrets.yaml && \
-                kubectl get secret gluu-jackrabbit-postgres-pass -n gluu -o yaml >> gluu_secrets.yaml
-                ```        
-        
-            
-            1. If this is a restore it likely means the helm deployment of Gluu is currupt. Delete the helm deployment of Gluu in preperation for a new fresh one:
-            
-                ```bash
-                helm delete <release-name> -n <gluu-namespace>
-                ```
-            
-            1. Create `gluu` `Secret` and `ConfigMap` from backup saved previosuly:
-            
-               ```bash
-               kubectl create -f gluu_main_secret.yaml && kubectl create -f gluu_main_cm.yaml
-               ```
-            
-            1. Run the install command for helm:
-            
-                ```bash
-                helm install <release-name> -f ./values.yaml . -n <gluu-namespace>
-                ```
-                
-            1. Move the backup ldifs to the new opendj pod at `/opt/opendj/ldif`
-            
-            1.  Choose the backup of choice and run the command below:
-            
-                ```bash
-                /opt/opendj/bin/import-ldif --hostname localhost --port 4444 --bindDN "cn=Directory manager" --backendID userRoot --trustAll --ldifFile /opt/opendj/ldif/backup-4.ldif --bindPassword "<Password>"
-                ```
-            
-            1. Preform a rolling update of each service, forexample :             
-            
-               ```bash
-               kubectl rollout restart gluu-deployment -n gluu
-               ```
+            /opt/opendj/bin/import-ldif --hostname localhost --port 4444 --bindDN "cn=Directory manager" --backendID userRoot --trustAll --ldifFile /opt/opendj/ldif/backup-1.ldif --bindPassword "<Password>"
+            ```                     
             
+    === "SQL"
+
+        SQL database backup is a common feature whether in a User-managed or Cloud-Managed deployment.
```

Comment:
can you add an example of executing a command to backup the database and restore it manually.